snippets/snippets.json

```diff
@@ -1,19 +1,143 @@
 {
-  "Console Log": {
-    "prefix": "clo",
+  "imr": {
+    "prefix": "imr",
+    "body": "import React from 'react'\n$1",
```

Comment:
I'm not a fan of enforcing new lines insertion on the user. Plus snippets get final tab stop added to its end by default, so there's no need for `$1`.